docs/architecture/adr-016-social-upgrades.md

```diff
@@ -19,12 +19,13 @@ It should also be noted that this ADR is not discussion parameter changes, as li
 ## Alternative Approaches
 
 ### Option 1: Temporarily use the current token voting mechanism
-
 Given that we don't have a lot of time until mainnet, we could leave the current implementation in place which gives us the option to use it if needed. Ideally, we would work on the longer term upgrade mechanism that respects social consensus and finish it before the first upgrade. While this option does allow for maximum flexibility, it is also very risky because if the mechanism is ever needed or used, then it could set a precedent for future upgrades.
 
-### Option 2: Predetermined halt height and removal of token voting for upgrades (aka "difficulty bomb")
+### Option 2: Removal of token voting for upgrades
+The goal of this approach is to force social consensus to reach an upgrade instead of relying on token voting. It works by simply removing the currently upgrade module. This way, the only way to upgrade the chain is to agree on the upgrade logic and the upgrade height.
```

Comment:
```suggestion
The goal of this approach is to force social consensus to reach an upgrade instead of relying on token voting. It works by simply removing the current upgrade module. This way, the only way to upgrade the chain is to agree on the upgrade logic and the upgrade height.
```